dsgrid/registry/project_registry_manager.py

```diff
@@ -764,12 +772,8 @@ def _submit_dataset_and_register_mappings(
     ):
         logger.info("Submit dataset=%s to project=%s.", dataset_id, project_config.config_id)
         self._check_if_not_registered(project_config.config_id)
+        self._raise_if_not_unregistered(project_config, dataset_id)
```

Comment:
Is this redundant with the line above?

Reply:
Line 774 checks if the project_id is registered. Line 775 checks if the dataset is present in the project config and has an unregistered status.

Reply:
I see that now--apologies for missing it the first time!